### ISSUE TITLE
3.0 - Upgrade Notice Revision Request

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7834]
### Explain your enhancement (please be detailed)
Currently, the 3.0 upgrade notice reads:

> Easy Digital Downloads needs to upgrade the database, click [here](#) to start the upgrade. [Learn more about this upgrade.](#)

Could we consider updating this notice to something which might look more like this?

![image](https://user-images.githubusercontent.com/4853066/81105652-3f103000-8ee2-11ea-8f7e-e3d8d7d43b49.png)

### Justification or use case
I have a few reasons to request this:
* grammar: as it currently stands, the first sentence is using a comma to join two independent clauses, but a semicolon is more appropriate
* the "action" link (to begin the upgrade) reads "here" which is not appropriately accessible link text
* the "action" link is before the informational link ("Learn more...")

I put together a screenshot because I think changing the "action" link to include the action being performed would be helpful. Additionally, the visual change to styling it as a button helps reinforce the idea that an action is about to be taken (although this link is technically just a link to another page, it is the beginning of an action sequence).

It may also be helpful to include at least some of the "learn more" text on the top of the migration page, in case a user navigates there without clicking the "learn more"...especially the note to back up the database and that the upgrade is irreversible.

**Comment by spencerfinnell** [comment]
I like the CTA button, but I believe it should be on its own line under the text. The right side of inline notices tend to be for dismiss buttons. It also brings a little more height to the notice, which I think will help it be seen -- we could also use the yellow `warning` notice styling.

**Comment by ashleyfae** [comment]
> I like the CTA button, but I believe it should be on its own line under the text. The right side of inline notices tend to be for dismiss buttons. It also brings a little more height to the notice, which I think will help it be seen -- we could also use the yellow `warning` notice styling.

Good point on being on its own line. Off to the right it could also get lost on really large screens (where the CTA button would be super far away from the associated text).

**Comment by cklosowski** [comment]
Along with this, I think I’d like to expose the existence of the CLI tool on the upgrade screen. And make it prominent.

In the past we’ve exposed that in a ‘Learn More’ link, but I think having someone click the CTA, and take them to the upgrade page (where they still do have to click once more) would be idea.

At that point we can adjust the verbiage at the top to include the CLI tool, and tell them that it is possible for this take long amount of time, dependent on the size of their store, etc, etc. Take backups, etc.

**Comment by ashleyfae** [comment]
> Along with this, I think I’d like to expose the existence of the CLI tool on the upgrade screen. And make it prominent.
> 
> In the past we’ve exposed that in a ‘Learn More’ link, but I think having someone click the CTA, and take them to the upgrade page (where they still do have to click once more) would be idea.
> 
> At that point we can adjust the verbiage at the top to include the CLI tool, and tell them that it is possible for this take long amount of time, dependent on the size of their store, etc, etc. Take backups, etc.

100% agree. Related: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7885

**Comment by spencerfinnell** [comment]
Related #7902
